src/Components/Artwork/SaveButton/useSaveArtwork.ts

```diff
@@ -38,6 +43,12 @@ export const useSaveArtwork = ({
                 id: artwork.id ?? "",
                 slug: artwork.slug ?? "",
                 isSaved: !isSaved,
+                collectorSignals: {
+                  auction: {
+                    lotWatcherCount:
+                      artwork.collectorSignals?.auction?.lotWatcherCount ?? 0,
+                  },
+                },
```

Comment:
Ah, now I see. github linked me to these files from outside the PR branch. makes more sense now. Would `collectorSignals: artwork.collectorSignals ?? null` accomplish the same thing?